ydb/types.py

```diff
@@ -12,12 +11,8 @@
 
 _SECONDS_IN_DAY = 60 * 60 * 24
 _EPOCH = datetime(1970, 1, 1)
-if six.PY3:
-    _from_bytes = None
-else:
 
-    def _from_bytes(x, table_client_settings):
-        return _utilities.from_bytes(x)
+_from_bytes = None
```

Comment:
it would be nice to remove `_from_bytes` at all

Reply:
great!